services/app-api/handlers/kafka/post/postKafkaData.ts

```diff
@@ -14,9 +14,36 @@ const tables = [
   { sourceName: reportTables.NAAAR, topicName: tableTopics.NAAAR },
 ];
 const buckets = [
-  { sourceName: reportBuckets.MCPAR, topicName: bucketTopics.MCPAR },
-  { sourceName: reportBuckets.MLR, topicName: bucketTopics.MLR },
-  { sourceName: reportBuckets.NAAAR, topicName: bucketTopics.NAAAR },
+  {
+    sourceName: reportBuckets.MCPAR,
+    topicName: bucketTopics.MCPAR,
+    s3Prefix: "fieldData/",
```

Comment:
This preps for the handling, but KafkaSourceLib has to then handle the filtering you provide it. You'll need to update that library.